site_root/input/ImplementationGuide-NCPI-Project-Forge.json

```diff
@@ -141,6 +141,62 @@
         "reference": {
           "reference": "StructureDefinition/age-at-event"
         }
+      },
+      {
+        "description": "NCPI Project Forge ValueSet phenotype-codes",
+        "exampleBoolean": false,
+        "name": "NCPI Project Forge ValueSet/phenotype-codes",
+        "reference": {
+          "reference": "ValueSet/phenotype-codes"
+        }
+      },
+      {
+        "description": "NCPI Project Forge CodeSystem hp",
+        "exampleBoolean": false,
+        "name": "NCPI Project Forge CodeSystem/hp",
+        "reference": {
+          "reference": "CodeSystem/hp"
+        }
+      },
+      {
+        "description": "NCPI Project Forge ValueSet hpo-observation-type-codes",
+        "exampleBoolean": false,
+        "name": "NCPI Project Forge ValueSet/hpo-observation-type-codes",
+        "reference": {
+          "reference": "ValueSet/hpo-observation-type-codes"
+        }
+      },
```

Comment:
I think this ValueSet can have a more global usage scope, not just for HPO. Maybe, just `"observation-type-codes"`?

Reply:
I chose the former option. That seems like a reasonable solution to me.